frontends/web/src/routes/buy/components/exchangeselectionradio.module.css

```diff
@@ -129,7 +129,7 @@
 /* checked */
 
 .radio[aria-checked="true"] {
-    background-color: var(--color-light-skyblue);
+    background-color: var(--background);
```

Comment:
in light mode `--background` is using `--color-lightgray`, but here it was `--color-light-skyblue` 🤔 

is this for focusing the background of the whole option?
do we need a new CSS variable `--background-focus` or similar?

Reply:
> is this for focusing the background of the whole option?
Yes exactly.

> do we need a new CSS variable --background-focus or similar?
That'd be good, I think. As lightmode uses skyblue and in darkmode it obviously won't. WDYT?

Reply:
makes sense to me

Reply:
I was testing it out and I think it is more intuitive for the focus background to be the lighter one. So like this:

![image](https://user-images.githubusercontent.com/42831937/225614292-f99a90e7-03db-4e58-8571-7027e3d89e33.png)

Reply:
agree, looks better too @jadzeidan . Thanks.